localisation/src/survey/calculations/index.ts

```
@@ -0,0 +1,87 @@
+        return address.rent;
+    }
+    case 'buy':
+        if (
+            typeof address.mortgage !== 'number' ||
+                typeof address.interestRate !== 'number' ||
+                typeof address.amortizationPeriod !== 'string'
+        ) {
+            console.error('Incomplete mortgage information for address when calculating monthly housing cost');
+            return null;
+        }
+        const amortizationPeriod = parseInt(address.amortizationPeriod!, 10);
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)